frontend/src/features/admin-form/settings/components/MrfFormEmailSection.tsx

```diff
@@ -0,0 +1,205 @@
+    (formData) => {
+      const selectedSteps = formData[WorkflowEmailMultiSelectName]
+      const selectedEmails = formData[OtherPartiesEmailInputName]
+
+      return handleSubmitEmailNotificationSettings({
+        nextStepsToNotify: selectedSteps,
+        nextStaticEmails: selectedEmails,
+      })
+    },
+    [handleSubmitEmailNotificationSettings],
+  )
+  const onSubmitDebounced = useCallback(
+    (formData) => debounce(() => onSubmit(formData), DEBOUNCE_DELAY_IN_MS)(),
+    [onSubmit],
+  )
+
+  const handleWorkflowEmailMultiSelectChange = useCallback(() => {
+    handleSubmit(onSubmitDebounced)()
+  }, [handleSubmit, onSubmitDebounced])
+
+  const handleOtherPartiesEmailInputBlur = useCallback(() => {
+    const uniqueValidEmails = uniq(
+      filterInvalidEmails(getValues(OtherPartiesEmailInputName)),
+    )
+    setValue(OtherPartiesEmailInputName, uniqueValidEmails)
+    handleSubmit(onSubmit)()
+  }, [getValues, handleSubmit, onSubmit, setValue, filterInvalidEmails])
+
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <Box>
+        <FormLabel>Notify respondents in your workflow</FormLabel>
+        <Skeleton isLoaded={!isLoading}>
+          <Box my="0.75rem">
+            <Controller
+              control={control}
+              name={WorkflowEmailMultiSelectName}
+              render={({
+                // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                field: { value: values = [], onChange, ...rest },
+              }) => (
+                <MultiSelect
+                  items={formWorkflowStepsWithStepNumber.map((step) => ({
+                    label: `Respondent(s) in Step ${step.stepNumber}`,
+                    value: step._id,
+                  }))}
+                  values={values}
+                  onChange={(values) => {
+                    onChange(values)
+                    handleWorkflowEmailMultiSelectChange()
+                  }}
+                  placeholder="Select respondents from your form"
+                  isSelectedItemFullWidth
+                  isDisabled={isLoading || isDisabled}
+                  {...rest}
+                />
+              )}
+            />
+          </Box>
+        </Skeleton>
+      </Box>
+      <Box my="2rem">
+        <FormControl
+          isInvalid={!isEmpty(errors[OtherPartiesEmailInputName])}
+          isDisabled={isDisabled}
+        >
+          <FormLabel
+            tooltipVariant="info"
+            tooltipPlacement="top"
+            tooltipText="Include the admin's email to inform them whenever a workflow is completed"
+          >
+            Notify other parties
+          </FormLabel>
+          <Controller
+            name={OtherPartiesEmailInputName}
+            control={control}
+            rules={OPTIONAL_ADMIN_EMAIL_VALIDATION_RULES}
+            render={({ field }) => (
+              <TagInput
+                {...(getValues(OtherPartiesEmailInputName)?.length > 0
```

Comment:
Could you help to improve the placeholder declaration with the below code? Understand that this is not related to your recent changes.

```suggestion
  // hide display of placeholder if there are values in the field
  const otherFieldPlaceholder =
    getValues(OtherPartiesEmailInputName)?.length > 0
      ? undefined
      : 'me@example.com'
    ...
      <TagInput
        placeholder={otherFieldPlaceholder}
        ...
      />
      
```